app.py

```diff
@@ -19,26 +18,49 @@ def index():
     return {"message": data}
 
 
-
 @app.route('/recommend/popular')
 def popular_movies():
     data = recommend_popular_movies()
     return data
 
 
```

Comment:
These two lines of code are placed outside of any function or route, which means they will throw an error. Please move them inside a route or function.